requests_oauthlib/oauth2_session.py

```diff
@@ -279,9 +297,7 @@ def refresh_token(self, token_url, refresh_token=None, body='', auth=None,
 
         refresh_token = refresh_token or self.token.get('refresh_token')
 
-        log.debug('Adding auto refresh key word arguments %s.',
-                  self.auto_refresh_kwargs)
-        kwargs.update(self.auto_refresh_kwargs)
+        log.debug('Kwargs: %s', kwargs)
```

Comment:
This log message isn't really useful anymore: it might be better to remove it rather than leave it like this.